src/main/java/com/example/Restaurant_Management_System_REST_API/rest/controller/InventoryItemController.java

```diff
@@ -0,0 +1,46 @@
+package com.example.Restaurant_Management_System_REST_API.rest.controller;
+
+import com.example.Restaurant_Management_System_REST_API.DTO.InventoryItemDTOs.InventoryItemDTORequest;
+import com.example.Restaurant_Management_System_REST_API.DTO.InventoryItemDTOs.InventoryItemDTOResponse;
+import com.example.Restaurant_Management_System_REST_API.exception.NotFoundInDatabaseException;
+import com.example.Restaurant_Management_System_REST_API.service.InventoryItemService;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/inventory/")
+@AllArgsConstructor
+public class InventoryItemController {
+
+    private InventoryItemService inventoryItemService;
```

Comment:
I'm marking another field in controller/service (across multiple PRs now) with not having `final` keyword. Do you forget about it or what's wrong here?

Reply:
I forgot to change it in this branch LOL